src/booleans.js

```diff
@@ -1,62 +1,65 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !a && !b;
+}
 
 function one(a, b) {
-  // your code here
-};
+  return (a && !b) || (!a && b);
+}
 
 function truthiness(a) {
-  // your code here
-};
+  if (a) {
+    return true;
+  }
+  return false;
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  return a === b;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  return a > b;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  return a <= b;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  return a % 2 !== 0;
+}
 
 function isEven(a) {
-  // your code here
-};
+  return a % 2 === 0;
+}
 
 function isSquare(a) {
-  // your code here
-};
+  return Math.sqrt(a) % 1 === 0;
+}
 
 function startsWith(char, string) {
-  // your code here
-};
+  return string.charAt(0) === char;
+}
 
 function containsVowels(string) {
-  // your code here
-};
+  return /[aeiouAEIOU]/.test(string);
```

Comment:
You can add an `i` after the last `/` in regex to make it case insensitive, so you don't have to put each character as capital and lowercase. i.e.
```
return /[aeiou]/i.test(string);
```